librz/core/cmd_remote.c

```diff
@@ -9,51 +9,51 @@
+	"R", "", "list all open connections",
+	"R<", "[fd] cmd", "send output of local command to remote fd", // XXX may not be a special char
+	"R", "[fd] cmd", "exec cmd at remote 'fd' (last open is default one)",
+	"R!", " cmd", "run command via rz_io_system",
+	"R+", " [proto://]host:port", "connect to remote host:port (*rap://, raps://, tcp://, udp://, http://)",
+	"R-", "[fd]", "remove all hosts or host 'fd'",
+	"R=", "[fd]", "open remote session with host 'fd', 'q' to quit",
+	"R!=", "", "disable remote cmd mode",
+	"R=!", "", "enable remote cmd mode",
 	"\nservers:", "", "",
 	".:", "9000", "start the tcp server (echo x|nc ::1 9090 or curl ::1:9090/cmd/x)",
-	"=:", "port", "start the rap server (o rap://9999)",
-	"=g", "[?]", "start the gdbserver",
-	"=h", "[?]", "start the http webserver",
-	"=H", "[?]", "start the http webserver (and launch the web browser)",
+	"R:", "port", "start the rap server (o rap://9999)",
+	"Rg", "[?]", "start the gdbserver",
+	"Rh", "[?]", "start the http webserver",
+	"RH", "[?]", "start the http webserver (and launch the web browser)",
 	"\nother:", "", "",
-	"=&", ":port", "start rap server in background (same as '&_=h')",
-	"=", ":host:port cmd", "run 'cmd' command on remote server",
+	"R&", ":port", "start rap server in background (same as '&_=h')",
+	"R", ":host:port cmd", "run 'cmd' command on remote server",
 	"\nexamples:", "", "",
-	"=+", "tcp://localhost:9090/", "connect to: rizin -c.:9090 ./bin",
-	// "=+", "udp://localhost:9090/", "connect to: rizin -c.:9090 ./bin",
-	"=+", "rap://localhost:9090/", "connect to: rizin rap://:9090",
-	"=+", "http://localhost:9090/cmd/", "connect to: rizin -c'=h 9090' bin",
+	"R+", "tcp://localhost:9090/", "connect to: rizin -c.:9090 ./bin",
+	// "R+", "udp://localhost:9090/", "connect to: rizin -c.:9090 ./bin",
+	"R+", "rap://localhost:9090/", "connect to: rizin rap://:9090",
+	"R+", "http://localhost:9090/cmd/", "connect to: rizin -c'=h 9090' bin",
 	"o ", "rap://:9090/", "start the rap server on tcp port 9090",
 	NULL
 };
 
 static const char *help_msg_equalh[] = {
 	"Usage:", " =[hH] [...]", " # http server",
 	"http server:", "", "",
-	"=h", " port", "listen for http connections (rizin -qc=H /bin/ls)",
-	"=h-", "", "stop background webserver",
-	"=h--", "", "stop foreground webserver",
-	"=h*", "", "restart current webserver",
-	"=h&", " port", "start http server in background",
-	"=H", " port", "launch browser and listen for http",
-	"=H&", " port", "launch browser and listen for http in background",
+	"Rh", " port", "listen for http connections (rizin -qc=H /bin/ls)",
+	"Rh-", "", "stop background webserver",
+	"Rh--", "", "stop foreground webserver",
+	"Rh*", "", "restart current webserver",
+	"Rh&", " port", "start http server in background",
+	"RH", " port", "launch browser and listen for http",
+	"RH&", " port", "launch browser and listen for http in background",
 	NULL
 };
 
 static const char *help_msg_equalg[] = {
 	"Usage:", " =[g] [...]", " # gdb server",
```

Comment:
Same here. Probably also it makes sense to rename the name of the variable.

Reply:
> Same here. Probably also it makes sense to rename the name of the variable.

@XVilka Thanks, and I change them.